decidim/decidim
# Fix Updating a Conference speaker with an invalid image, will throw error

#### :tophat: What? Why?
 Updating a Conference Speaker with a picture with more that 3840px, will raise an ActiveRecord::RecordInvalid, as the the conference speaker is created using Decidim.traceability.update! .

Additional details on #7188 
#### :pushpin: Related Issues
- Fixes #7188

#### Testing

1. Go to Conference speaker admin section
2. Edit a speaker
3. Go to the image upload
4. Attach the map image from this ticket
5. See the error
6. Apply the patch
7. See the error will become a form error

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](../CONTRIBUTING.adoc) to this repository.


### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
![Description](URL)

:hearts: Thank you!
